jiant/scripts/download_data/dl_datasets/files_tasks.py

```diff
@@ -170,6 +194,219 @@ def download_abductive_nli_data_and_write_config(
+            os.path.join(task_data_path, file_name),
+        )
+    py_io.write_json(
+        data={
+            "task": task_name,
+            "paths": {
+                "train": os.path.join(task_data_path, "train.tsv"),
+                "val": os.path.join(task_data_path, "val.tsv"),
+                "test": os.path.join(task_data_path, "test.tsv"),
+                "train_doubled": os.path.join(task_data_path, "train-doubled.tsv"),
+                "train_w_swap": os.path.join(task_data_path, "train-w-swap.tsv"),
+                "train_w_swap_doubled": os.path.join(task_data_path, "train-w-swap-doubled.tsv"),
+            },
+            "name": task_name,
+        },
+        path=task_config_path,
+    )
+
+
+def download_mctaco_data_and_write_config(
+    task_name: str, task_data_path: str, task_config_path: str
+):
+    os.makedirs(task_data_path, exist_ok=True)
+    file_name_list = ["dev_3783.tsv", "test_9442.tsv"]
+    for file_name in file_name_list:
+        download_utils.download_file(
+            f"https://raw.githubusercontent.com/CogComp/MCTACO/master/dataset/{file_name}",
+            os.path.join(task_data_path, file_name),
+        )
+    py_io.write_json(
+        data={
+            "task": task_name,
+            "paths": {
+                "val": os.path.join(task_data_path, "dev_3783.tsv"),
+                "test": os.path.join(task_data_path, "test_9442.tsv"),
+            },
+            "name": task_name,
+        },
+        path=task_config_path,
+    )
+
+
+def download_mctest160_data_and_write_config(
+    task_name: str, task_data_path: str, task_config_path: str
+):
+    os.makedirs(task_data_path, exist_ok=True)
+    download_utils.download_and_unzip(
+        "https://mattr1.github.io/mctest/data/MCTest.zip", task_data_path,
+    )
+    download_utils.download_and_unzip(
+        "https://mattr1.github.io/mctest/data/MCTestAnswers.zip", task_data_path,
+    )
+    os.rename(
+        os.path.join(task_data_path, "MCTestAnswers", f"mc160.test.ans"),
+        os.path.join(task_data_path, "MCTest", f"mc160.test.ans"),
+    )
+    shutil.rmtree(os.path.join(task_data_path, "MCTestAnswers"))
+    for phase in ["train", "dev", "test"]:
+        os.rename(
+            os.path.join(task_data_path, "MCTest", f"mc160.{phase}.tsv"),
+            os.path.join(task_data_path, f"mc160.{phase}.tsv"),
+        )
+        os.rename(
+            os.path.join(task_data_path, "MCTest", f"mc160.{phase}.ans"),
+            os.path.join(task_data_path, f"mc160.{phase}.ans"),
+        )
+    shutil.rmtree(os.path.join(task_data_path, "MCTest"))
+
+    py_io.write_json(
+        data={
+            "task": task_name,
+            "paths": {
+                "train": os.path.join(task_data_path, "mc160.train.tsv"),
+                "train_ans": os.path.join(task_data_path, "mc160.train.ans"),
```

Comment:
Is the following suitable for the task? I want to try and reduce the number of named keys in a task config (`*_ans` is a new key).
```suggestion
                "train_labels": os.path.join(task_data_path, "mc160.train.ans"),
```